lasagne/utils.py

```diff
@@ -249,6 +249,9 @@ def create_param(spec, shape, name=None):
     support initialization with numpy arrays, existing Theano shared
     variables, and callables for generating initial parameter values.
     """
+    if any([d == 0 for d in shape]):
```

Comment:
You can omit the square brackets to make it a generator expression instead of a list comprehension.